### PR TITLE
Remove unused conversation response code

### PR DESCRIPTION
**Pull request by MrPetovan**
Address https://github.com/friendica/friendica/issues/8000#issuecomment-587138376
Address https://github.com/friendica/friendica/issues/8000#issuecomment-589263329

This PR touches a part of Friendica I've always wanted to redo because it's hard to understand but it was working so I didn't: the conversation responses. Until today, when this notice has been the perfect pretext to cull it to its bare use. It still is unsatisfactory, but it is less of a code wart now.

### CODE REVIEW

File: include/conversation.php [hunk 1]
```diff
@@ -579,12 +579,11 @@ function conversation(App $a, array $items, $mode, $update, $preview = false, $o
 	$items = $cb['items'];
 
 	$conv_responses = [
-		'like' => ['title' => DI::l10n()->t('Likes','title')],
-		'dislike' => ['title' => DI::l10n()->t('Dislikes','title')],
-		'attendyes' => ['title' => DI::l10n()->t('Attending','title')],
-		'attendno' => ['title' => DI::l10n()->t('Not attending','title')],
-		'attendmaybe' => ['title' => DI::l10n()->t('Might attend','title')],
-		'announce' => ['title' => DI::l10n()->t('Reshares','title')]
```
Comment (AlfredSK) [inline include/conversation.php]: @MrPetovan Looks like you removed the 'announce' response. Is this done on purpose or by accident?

Comment (MrPetovan) [reply include/conversation.php]: Probably by accident, do reshare counts don't show anymore?

Comment (AlfredSK) [reply include/conversation.php]: Exactly! I just wanted to do the bugfix PR after I found the solution by re-adding a line for 'announce'. :-)

Comment (MrPetovan) [reply include/conversation.php]: You're getting very good at this!

**Pull request merged**